src/PrettierEditProvider.ts

```diff
@@ -22,6 +22,7 @@ import {
     ParserOption,
 } from './types.d';
 
+const bundledPrettier = require('prettier') as Prettier;
```

Comment:
I would say laziness with typings now. Before it could be to defer the loading, but that's not the case anymore.